File: SDKs/googleapis.com/sqladmin/v1/java/lib/src/main/java/openapisdk/models/shared/SslCertsCreateEphemeralRequest.java
package openapisdk.models.shared;

import com.fasterxml.jackson.annotation.JsonProperty;
import com.fasterxml.jackson.annotation.JsonInclude;
import com.fasterxml.jackson.annotation.JsonInclude.Include;

/**
 * SslCertsCreateEphemeralRequest
 * SslCerts create ephemeral certificate request.
**/
public class SslCertsCreateEphemeralRequest {
    @JsonInclude(Include.NON_ABSENT)
    @JsonProperty("access_token")
    public String accessToken;
    public SslCertsCreateEphemeralRequest withAccessToken(String accessToken) {
        this.accessToken = accessToken;
        return this;
    }
    @JsonInclude(Include.NON_ABSENT)
    @JsonProperty("public_key")
    public String publicKey;
    public SslCertsCreateEphemeralRequest withPublicKey(String publicKey) {
        this.publicKey = publicKey;
        return this;
    }
}

File: SDKs/googleapis.com/sql/v1beta4/python/src/sdk/models/shared/sslcertscreateephemeralrequest.py
import dataclasses
from typing import Optional
from dataclasses_json import dataclass_json
from sdk import utils


@dataclass_json
@dataclasses.dataclass
class SslCertsCreateEphemeralRequest:
    r"""SslCertsCreateEphemeralRequest
    SslCerts create ephemeral certificate request.
    """
    
    access_token: Optional[str] = dataclasses.field(default=None, metadata={'dataclasses_json': { 'letter_case': utils.field_name('access_token') }})
    public_key: Optional[str] = dataclasses.field(default=None, metadata={'dataclasses_json': { 'letter_case': utils.field_name('public_key') }})
    
